spec/classes/profile/openvpn_spec.rb

```diff
@@ -29,3 +29,34 @@
   it { expect(subject).to contain_firewall "100 allow routing from 172.19.0.0/24 to 10.0.0.0/16 on ports 80/443" }
   it { expect(subject).to contain_exec "addroute 10.0.0.0 through 192.168.100.1 (NIC eth1)" }
 end
+
+
+describe 'profile::openvpn' do
+  let(:facts) do
+    {
+      :rspec_hieradata_fixture => 'profile_openvpn_two_interfaces',
+    }
+  end
+
+  it { expect(subject).to contain_class 'stdlib' }
+  it { expect(subject).to contain_class 'profile::docker' }
+
+  it { expect(subject).to contain_package 'net-tools' }
+
+  it { expect(subject).to contain_file '/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg' }
```

Comment:
```suggestion
  it { expect(subject).not_to contain_file '/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg' }
```

as per https://github.com/rspec/rspec-expectations/tree/v3.11.0#equivalence

Reply:
`.not_to` fails (as `should_not`):

```
Failures:

  1) profile::openvpn with 2 interfaces is expected not to contain File[/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg]
     Failure/Error: it { expect(subject).not_to contain_file '/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg' }
     
     Puppet::ParseErrorWithIssue:
       Could not parse for environment rp_env: This Name has no effect. A value was produced and then forgotten (one or more preceding expressions may have the wrong form) (line: 115, column: 26) on node macbook-pro-de-herve.local
```